impl/src/main/java/org/pentaho/di/trans/dataservice/streaming/StreamServiceKey.java

```diff
@@ -2,7 +2,7 @@
  *
  * Pentaho Data Integration
  *
- * Copyright (C) 2018 by Hitachi Vantara : http://www.pentaho.com
+ * Copyright (C) 2002-2018 by Hitachi Vantara : http://www.pentaho.com
```

Comment:
I believe this is a new file, why put 2002 in the license header?

Reply:
Indeed, the PR was updated.